tests/acceptance/helpers/sharingHelper.js

```diff
@@ -156,10 +156,9 @@ module.exports = {
           'Could not get shares. Message: ' + sharesResult.ocs.meta.message
         )
         const shares = sharesResult.ocs.data
-        let lastFoundShareId = 0
+        const lastShareSTime = 0
         for (const share of shares) {
-          if (share.share_type === self.SHARE_TYPES.public_link && share.id >= lastFoundShareId) {
-            lastFoundShareId = share.id
+          if (share.share_type === self.SHARE_TYPES.public_link && share.stime > lastShareSTime) {
```

Comment:
I think you need to update the lastShareSTime on the loop every time.
Otherwise it will always be 0

Reply:
> I think you need to update the lastShareSTime on the loop every time.
> Otherwise it will always be 0
Silly mistake